src/pkcs11/mechanism.c

```diff
@@ -1300,6 +1342,16 @@ void free_info(const void *info)
 	free((void *) info);
 }
 
+CK_RV copy_hash_signature_info(const void *mech_data, void **new_data)
+{
+	*new_data = calloc(1, sizeof(struct hash_signature_info));
```

Comment:
please add a check for `mech_data != NULL`

Reply:
Fixed